basedtyping/internal_typing_stubs.py

```diff
@@ -0,0 +1,90 @@
+"""stubs for the internal "type machinery" defined in the typing module.
```

Comment:
What's the purpose of this?

Reply:
so that `_GenericAlias` is typed. alternative is suppressing a ton of mypy errors in `reified_generic.py`, i figured it's better to just type these internal things so they're easier to use

Reply:
need to move `internal_typing_stubs` to basedtypeshed but can't because idk how to use basedtypeshed in this project. https://github.com/KotlinIsland/basedtypeshed/pull/3

Reply:
ive made it private for now and added documentation saying it will be moved

Reply:
moved it to basedmypy's typeshed https://github.com/KotlinIsland/basedmypy/pull/131